Project.toml

```diff
@@ -5,6 +5,13 @@ version = "1.0.0-DEV"
 
 [deps]
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+Debugger = "31a5f54b-26ea-5ae9-a837-f05ce5417438"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
+OhMyREPL = "5fb14364-9ced-5910-84b2-373655c76a03"
+PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
we shouldn't have these packages in the dependency of `AwkwardArrays.jl`.

For "developer package", usually people can leave them in the home environment.

In Julia, the home environment stacks, so even you're developing `AwkwardArray.jl` by doing
```
julia --project=.
```

you can still `using Revise...` if you home environment has `Revise`